pkg/entrypoint/entrypointer_test.go

```diff
@@ -204,6 +213,39 @@ func TestEntrypointer(t *testing.T) {
 	}
 }
 
+func assertExist(t *testing.T, entries []v1alpha1.PipelineResourceResult, key ...string) {
```

Comment:
In general I don't like assert-style helpers in Go tests, it can obscure whether the helper Errors or Fatals. Can we inline this instead?

Reply:
yes , of course, I will change it.